### PR TITLE
Update admin styling to differences from the design

### PR DESCRIPTION
**Pull request by JPry**
### 🎫 Ticket

[TICKET_ID]

### 🎥 Artifacts 

Updated style using this branch:
Screenshot 2024-08-21 at 23 05 46

Screenshot 2024-08-21 at 23 43 53

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/resources/postcss/tribe-common-admin/settings/_props.pcss [hunk 1]
```diff
@@ -21,4 +21,7 @@
 	/** new styles to go in utilities */
 	--tec-border-radius-settings: 6px;
 	--tec-width-settings-form: 900px;
+
+	/** Font styles */
+	--tec-font-family-arial: Arial, sans-serif;
```
Comment (Camwyn) [inline src/resources/postcss/tribe-common-admin/settings/_props.pcss:26]:
```suggestion
```

nope - we have one, let's use it:
`--tec-font-family-sans-serif: "Helvetica Neue", Helvetica, -apple-system, BlinkMacSystemFont, Roboto, Arial, sans-serif;`

I don't want to add a new font family - this one is well tested and covers a lot of edge cases.

File: src/resources/postcss/tribe-common-admin/settings/_page.pcss [hunk 2]
```diff
@@ -40,6 +40,9 @@
 
 			h1 {
 				background-color: var(--tec-color-background);
+				font-family: var(--tec-font-family-arial);
```
Comment (Camwyn) [inline src/resources/postcss/tribe-common-admin/settings/_page.pcss:43]:
```suggestion
				font-family: var(--tec-font-family-sans-serif);
```

File: src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss [hunk 3]
```diff
@@ -36,8 +37,9 @@
 	}
 
 	.tec-nav__tab {
-		font-family: var(--tec-font-family-sans-serif);
+		font-family: var(--tec-font-family-arial);
```
Comment (Camwyn) [inline src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss:40]: Don't change this
```suggestion
		font-family: var(--tec-font-family-sans-serif);
```

File: src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss [hunk 4]
```diff
@@ -193,7 +195,11 @@
 			}
 
 			.tec-nav__subnav {
+				border-inline: 1px solid var(--tec-admin-core-grey-10);
+				border-top: 1px solid var(--tec-admin-core-grey-10);
 				left: var(--tec-spacer-7);
+				/* This calc is to remove each of the 1px borders */
+				max-width: calc(var(--tec-width-settings-form) - 2px);
```
Comment (Camwyn) [inline src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss:202]:
```suggestion
				box-sizing: border-box;
				max-width: var(--tec-width-settings-form);
```

Comment (Camwyn) [reply src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss:202]: (and remove the comment)

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: src/resources/postcss/tribe-common-admin/_main.pcss [hunk 5]
```diff
@@ -189,6 +189,8 @@ table.plugins {
 
 .tribe-settings-form {
 	max-width: 1000px;
+	border-inline: 1px solid var(--tec-admin-core-grey-10);
+	border-bottom: 1px solid var(--tec-admin-core-grey-10);
```
Comment (Camwyn) [inline src/resources/postcss/tribe-common-admin/_main.pcss:193]: Why no border top?
```suggestion
	border: 1px solid var(--tec-admin-core-grey-10);
```

Comment (Camwyn) [reply src/resources/postcss/tribe-common-admin/_main.pcss:193]: Then you dan't have to mess with borders on the `<li>`

File: src/resources/postcss/tribe-common-admin/settings/_props.pcss (hunk 1 above)
Comment (JPry) [reply src/resources/postcss/tribe-common-admin/settings/_props.pcss:26]: This was to match the design, where the font was specifically supposed to be Arial.

File: src/resources/postcss/tribe-common-admin/_main.pcss (hunk 5 above)
Comment (JPry) [reply src/resources/postcss/tribe-common-admin/_main.pcss:193]: > Then you don't have to mess with borders on the `<li>`

The `<li>` is positioned on top of this element with `position: absolute`. This seems to cover the upper borders of the `.tribe-settings-form` element.

With some testing, it looks like we need the full border on `.tribe-settings-form`, and also the borders on the `<li>`. Here's a screenshot with *only* the form borders:

Screenshot 2024-08-21 at 21 19 14

Here's a screenshot with the `<li>` borders added:

Screenshot 2024-08-21 at 21 20 06

File: src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss (hunk 4 above)
Comment (JPry) [reply src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss:202]: We actually need this to be done via `calc()`. I think the `absolute` positioning of this element is causing the border to be added to the element width, and if you look closely you can see the edges don't align.

Here's a screenshot with the suggested change in place:

Screenshot 2024-08-21 at 21 20 06

The 2px extra length is visible on the right side.

Comment (JPry) [reply src/resources/postcss/tribe-common-admin/settings/_tabbed-nav.pcss:202]: 🤦🏻‍♂️ Nevermind, I overlooked the addition of the `box-sizing` style.

**Review by Camwyn — APPROVED** [review]
